### ISSUE TITLE
[Feature request] add names to the POIs

### ISSUE THREAD

**Issue by katerynakocherhina** [osmandapp/Osmand#8716]
If searching for the required POI directly on the map, it may be useful to see their name directly on the map without additionally tapping on them.

**Comment by scaidermern** [comment]
Already exists: Configure map -> POI overlay labels.

**Comment by FrankGail** [comment]
I mean the map markers i had create. I want to see their names on the card.

**Comment by vshcherb** [comment]
Yes, we can display POI names more selectively especially for 1 item instead of all items
